README.md

```diff
@@ -596,6 +596,7 @@ Options:
   * Should probably run the build in a docker container anyway
 * `yarn install`
 * `yarn build`
+  * Please note that you will need a Mapbox token in order to successfully run `yarn build`. You can get your own token [here](https://account.mapbox.com/access-tokens). Once you have a token, you can build the project by running `MAPBOX_ACCESS_TOKEN=your-token-here yarn build`.
```

Comment:
```suggestion
  * Please note that you will need a Mapbox token in order to successfully run `yarn build`, otherwise the build will fail due to failing to load the map images. You can get your own token [here](https://account.mapbox.com/access-tokens). Once you have a token, you should assign it to the `MAPBOX_ACCESS_TOKEN` environment variable.
```